extensions/ohif-cornerstone-extension/package.json

```diff
@@ -32,14 +32,14 @@
     "react-dom": "^15.0.0 || ^16.0.0",
     "react-redux": "^6.0.0",
     "react-resize-detector": "^3.4.0",
-    "react-viewerbase": "^0.2.10",
+    "react-viewerbase": "^0.6.0",
     "redux": "^4.0.1"
   },
   "dependencies": {
     "@babel/runtime": "^7.2.0",
     "classnames": "^2.2.6",
     "lodash.throttle": "^4.1.1",
-    "react-cornerstone-viewport": "0.1.29"
+    "react-cornerstone-viewport": "0.1.28"
```

Comment:
Downgrading?

Reply:
Not on purpose. My local version say this was the value that was last set:

![image](https://user-images.githubusercontent.com/5797588/58648783-7b9dd980-82d8-11e9-8e24-874d6922f8a7.png)

Possible I blew it out when merging in changes from master. I'll re-bump

Reply:
Marking this resolved. Bumped versions and cut new releases.